lib/packwerk/commands.rb

```diff
@@ -0,0 +1,30 @@
+# typed: true
+# frozen_string_literal: true
+
+module Packwerk
+  module Commands
+    extend ActiveSupport::Autoload
+
+    autoload :BaseCommand
+    autoload :CheckCommand
+    autoload :HelpCommand
+    autoload :InitCommand
+    autoload :UpdateCommand, "packwerk/commands/update_todo_command"
```

Comment:
Maybe we should add some aliasing functionality to make this more natural and explicit? i.e. here we're aliasing `update` to `update_todo`, so rather than adding an autoload we could have `execute_command` look at a list of aliases and "redirect" the command accordingly if an alias exists.

Or do we also need to support the existence of the `UpdateComand` constant as well? 🤔

Reply:
> Or do we also need to support the existence of the `UpdateCommand` constant as well? 🤔

We don't. It's a class introduced in this PR, but doesn't have to.

My problem is as follows:
- we use autoloading, so the class is not loaded _until_ we have defined that it's necessary.
- if we implement a system to declare aliases, then we need it to be run during initialization
- I would like to avoid having to declare aliases _outside_ of the command class itself (open-closed principle)

I guess the fact that we need these `autoload` is already an infringement to the open-closed principle though... 🤔 

Check this commit to see an early idea I had, where each command was able to declare itself, and its aliases in the process. The caveat was that command classes needed to be eager loaded.